nwcsafpps_runner/tests/test_utils.py

```diff
@@ -50,46 +49,45 @@ def create_files(d, typ):
         #: These files should not be found although the start time is correct
         f3 = d / "S_NWC_CMAPROB_noaa15_54321_19810305T0715000Z_19810305T0730000Z.{}".format(typ)
         f3.write_text("wrong orbit and correct time")
-        #: TODO: Create testfile older than 90 min but with correct orbitnumber
 
     #: Test xml files without start time
     typ = "xml"
     create_files(d, typ)
-    expected = ["{}/S_NWC_CMAPROB_noaa15_12345_19810305T0715000Z_19810305T0730000Z.{}".format(d.__str__(), typ),
-                "{}/S_NWC_CMAPROB_noaa15_12345_19810305T0745000Z_19810305T0800000Z.{}".format(d.__str__(), typ)]
+    expected = [os.path.join(d.__str__(), "S_NWC_CMAPROB_noaa15_12345_19810305T0715000Z_19810305T0730000Z.{}".format(typ)),
+                os.path.join(d.__str__(), "S_NWC_CMAPROB_noaa15_12345_19810305T0745000Z_19810305T0800000Z.{}".format(typ))]
```

Comment:
ok, if you have to do `__str__` I suppose `d` is a pathlib object ? in which case is more natural to use the `/` operator:
```suggestion
    expected = [d / "S_NWC_CMAPROB_noaa15_12345_19810305T0715000Z_19810305T0730000Z.{}".format(typ)),
                d / "S_NWC_CMAPROB_noaa15_12345_19810305T0745000Z_19810305T0800000Z.{}".format(typ))]
```

Reply:
Now I just did  
`os.path.join(d, "S_NWC_CMAPROB_noaa15_12345_19810305T0745000Z_19810305T0800000Z.{}".format(typ))]`
instead. Bt maybe `/` is better?

Reply:
It's fine as it is!